cis194/week2/zehua/src/Exercises.hs

```diff
@@ -1,7 +1,8 @@
 module Exercises where
 
-import Text.Read
-import Log
+import           Data.Maybe
+import           Log
+import           Text.Read
```

Comment:
does Haskell import order differentiate between system modules and user-defined modules? e.g., all system ones first then user-defined ones, like in PEP8 for python.

```
import           Data.Maybe
import           Text.Read

import           Log
```

Reply:
Conventionally, yes.
Also, it's clearer these years that it's better to import `qualified` or specify functions that needed. e.g.
```Haskell
import Data.Maybe qualified as Maybe
-- or
import Data.Maybe (fromJust)
```

Reply:
updated